{"timestamp":"2025-08-16T16:18:47.826429","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-08-16T16:18:47.830443","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/ai_models_etl.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-08-16T16:18:50.145583","level":"warning","event":"/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py:471: DeprecationWarning: Using Connection.get_connection_from_secrets from `airflow.models` is deprecated.Please use `from airflow.sdk import Connection` instead\n  warnings.warn(\n","logger":"py.warnings"}
{"timestamp":"2025-08-16T16:18:50.160877","level":"info","event":"Connection Retrieved 'models_connection'","logger":"airflow.hooks.base"}
{"timestamp":"2025-08-16T16:18:50.175712","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"OperationalError","exc_value":"connection to server at \"postgres\" (172.19.0.3), port 5432 failed: FATAL:  database \"models_db\" does not exist\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":890,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1177,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":217,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":240,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/ai_models_etl.py","lineno":110,"name":"load_to_postgres"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":763,"name":"run"},{"filename":"/usr/local/lib/python3.12/contextlib.py","lineno":137,"name":"__enter__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":870,"name":"_create_autocommit_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/hooks/postgres.py","lineno":183,"name":"get_conn"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py","lineno":122,"name":"connect"}],"is_group":false,"exceptions":[]}]}
